src/Pages/EmailList/EmailPage.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import './email-page.css';
+import EmailList from './EmailList';
+import EmailTemplate from './EmailTemplate';
+import Header from '../../Components/Header/Header';
+import {Row, TabContent, TabPane, Nav, NavItem, NavLink} from 'reactstrap';
+
+export default class EmailPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeTab: '1',
```

Comment:
whats '1' or '2'?

Reply:
'1' and '2' are for the different tabs on the page. So it starts out as '1' which is the 'Email List' tab, and '2' would be the 'Send Emails' tab. 
![image](https://user-images.githubusercontent.com/25803515/89138608-fbcd5980-d4f0-11ea-8707-7dd73647cb7f.png)

Reply:
can you change it to something more descriptive? e.g. `email-list and `send-emails`? will help others who will work on it in the future